python/grass/jupyter/init_funcs.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import grass.script as gs
+import grass.script.setup as gsetup
+
+
+def _set_notebook_defaults():
+    """
+    This function sets several GRASS environment variables that are
+    important for GRASS to run smoothly in Jupyter. A complete list and
+    description of environment variables can be found here:
+    https://grass.osgeo.org/grass78/manuals/variables.html
+    """
+    # We want functions to raise exceptions and see standard output of
+    # the modules in the notebook.
+    gs.set_raise_on_error(True)
+    gs.set_capture_stderr(True)
+
+    # Allow overwrite of existing maps
+    os.environ["GRASS_OVERWRITE"] = "1"
+
+
+def init(path, location=None, mapset=None):
```

Comment:
If you are not going to handle the more fancy parameters here, delete the default values. Both location and mapset are mandatory form grass.script.setup.init, so they need to be mandatory here to when you are just passing them through. The name `path` can stay as is. It makes sense even in this context.